app/lib/submission_builder/ty2022/states/az/individual_return.rb

```diff
@@ -87,7 +87,7 @@ def documents_wrapper
                     end
                     xml.RelationShip dependent.relationship
                     xml.NumMonthsLived dependent.months_in_home
-                    if dependent.dob <= 65.years.ago # TODO: needs to be based on a specific tax year date, also assumes we will have dob at all
+                    if dependent.dob <= MultiTenantService.statefile.end_of_current_tax_year.years_ago(65)
```

Comment:
unrelated to the prior last names change, but this was incorrectly using today's date instead of the end of the tax year